diff --git a/cursor_tail.glsl b/cursor_tail.glsl
index 5a648cc..6a5379d 100644
--- a/cursor_tail.glsl
+++ b/cursor_tail.glsl
@@ -4,10 +4,10 @@ vec3 sRGBToLinear(vec3 c) {
 }
 
 // --- CONFIGURATION ---
-vec4 TRAIL_COLOR = vec4(sRGBToLinear(iCurrentCursorColor.rgb), iCurrentCursorColor.a); // for custom color: vec4(0.2, 0.6, 1.0, 0.5); (wrap in sRGBToLinear for correct brightness)
-const float DURATION = 0.09; // in seconds
-const float MAX_TRAIL_LENGTH = 0.2;
-const float THRESHOLD_MIN_DISTANCE = 1.5; // min distance to show trail (units of cursor width)
+vec4 TRAIL_COLOR = iCurrentCursorColor; // for custom color: vec4(0.2, 0.6, 1.0, 0.5); (wrap in sRGBToLinear for correct brightness)
+const float DURATION = 0.2; // in seconds
+const float MAX_TRAIL_LENGTH = 0.5;
+const float THRESHOLD_MIN_DISTANCE = 0.0; // min distance to show trail (units of cursor width)
 const float BLUR = 2.0; // blur size in pixels (for antialiasing)
 
 // --- CONSTANTS for easing functions ---
@@ -30,14 +30,13 @@ const float SPRING_DAMPING = 0.9;
 // float ease(float x) {
 //     return 1.0 - (1.0 - x) * (1.0 - x);
 // }
 
-// // EaseOutCubic
-// float ease(float x) {
-//     return 1.0 - pow(1.0 - x, 3.0);
-// }
-
+// EaseOutCubic
+float ease(float x) {
+    return 1.0 - pow(1.0 - x, 3.0);
+}
 
 // // EaseOutQuart
 // float ease(float x) {
 //     return 1.0 - pow(1.0 - x, 4.0);
 // }
@@ -57,10 +56,10 @@ const float SPRING_DAMPING = 0.9;
 // float ease(float x) {
 //     return x == 1.0 ? 1.0 : 1.0 - pow(2.0, -10.0 * x);
 // }
 
-// EaseOutCirc
-float ease(float x) {
-    return sqrt(1.0 - pow(x - 1.0, 2.0));
-}
+// // EaseOutCirc
+// float ease(float x) {
+//     return sqrt(1.0 - pow(x - 1.0, 2.0));
+// }
 
 // // EaseOutBack
 // float ease(float x) {
